Gemfile

```diff
@@ -23,7 +23,7 @@ gem 'sidekiq-symbols'
 
 gem 'typhoeus'
 gem 'nokogiri'
-gem 'ox'
+gem 'ox', '2.4.12'
```

Comment:
Ta povodna (2.4.3) mi nesla skompilovat a najnovsia (2.4.13) ma [nejake zvlastnosti](https://github.com/ohler55/ox/issues/171) s whitespaceom. Ak je to problem, mozem sa na to neskor pozriet a nechat to bez verzie.